Ginger/GingerCoreNET/BPMN/ActivitiesGroupToBPMNConverter.cs

```diff
@@ -132,23 +178,30 @@ private IEnumerable<Activity> GetActivitiesFromActivityGroup(ActivitiesGroup act
                     .Select(identifier => identifier.IdentifiedActivity);
         }
 
-        private string GetTargetAppNameFromConsumerId(Consumer consumer)
+        private TargetBase GetTargetAppFromConsumer(Consumer consumer)
         {
             IEnumerable<TargetBase> targetApps = WorkSpace.Instance.Solution.GetSolutionTargetApplications();
             TargetBase? consumerTargetApp = targetApps.FirstOrDefault(targetApp => string.Equals(targetApp.Guid, consumer.ConsumerGuid));
             if (consumerTargetApp == null)
             {
-                throw new InvalidOperationException($"No Target Application found for Consumer with Guid '{consumer.ConsumerGuid}'.");
+                throw new BPMNExportException($"No Target Application found for Consumer with Guid '{consumer.ConsumerGuid}'.");
```

Comment:
Use Ginger Resource dictionary to show entity names according to the selected terminology